js_assets/Question.vue

```diff
@@ -110,6 +106,15 @@
                 })
                 this.answerVal = "";
             },
+            disableAnswer(questionPackage, index, answerVal) {
```

Comment:
What if we moved this to a calculated value?  If this.answerVal is not blank, set disabled to true else set it to false.  This way, everything is based on a single value (this.answerVal).

Reply:
cool! is computed disableAnswer in 9edc2a2 better?